burn-train/src/metric/dashboard/cli_stub.rs

```diff
@@ -1,11 +1,14 @@
-use crate::metric::dashboard::{DashboardMetricState, DashboardRenderer, TrainingProgress};
+use crate::{
+    metric::dashboard::{DashboardMetricState, DashboardRenderer, TrainingProgress},
+    TrainingInterrupter,
+};
 
 /// A simple renderer for when the cli feature is not enabled.
 pub struct CLIDashboardRenderer;
 
 impl CLIDashboardRenderer {
     /// Create a new instance.
-    pub fn new() -> Self {
+    pub fn new(_interuptor: TrainingInterrupter) -> Self {
         Self {}
     }
```

Comment:
I think the problem is that the renderer type is selected with feature flag over the type, where I think it should be when creating. I think we should have a function like the following:

```rust
pub fn rendered_default(interuptor: TrainingInterrupter) -> Rendered {
}
```